wiki/BBCode/en.md

```diff
@@ -1,581 +1,417 @@
-</tr>
-<tr class="even">
-<td><p>Note</p></td>
-<td><p><strong>Only works within the <a href="http://osu.ppy.sh/forum/12">Beatmaps</a> forum!</strong><br />
-For a weird reason, it also doesn't work in previews but works after having posted the message.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Syntax</p></td>
-<td><p>Write this</p></td>
-</tr>
-<tr class="even">
-<td><pre><code>[Your text here]</code></pre></td>
-<td><div style="color: rgb(67, 63, 117); margin: 1px 3px 5px 0px; padding-left: 8px; border-bottom: 1px solid rgb(190, 187, 205); box-shadow: 0px 3px 2px -3px rgb(190, 187, 205); font-size: 140%; font-weight: bold;">
-<p>Your text here</p>
-</div></td>
-</tr>
-</tbody>
-</table>
-
-White box
----------
-
-<table>
-<thead>
-<tr class="header">
-<th><p>Purpose</p></th>
-<th><p>Use this to insert a white box in which you can write text. Primarily used for design purposes.</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Dedicated button?</p></td>
-<td><p>No: you have to type this by hand.</p></td>
-</tr>
-<tr class="even">
-<td><p>Syntax</p></td>
-<td><p>Write this</p></td>
-</tr>
-<tr class="odd">
-<td><pre><code>[notice]A first line
-A second line[/notice]</code></pre></td>
-<td><div style="background: none repeat scroll 0% 0% rgb(249, 247, 254); border: 1px solid rgb(225, 223, 231); margin: 6px; padding: 5px;">
-<p>A first line<br />
-A second line</p>
-</div></td>
-</tr>
-</tbody>
-</table>
+# BBCode
+
+![The edit box in the forums](img/fourms.jpg "The edit box in the forums")
+
+**BBCode** is a markup syntax that is used in the osu!forums and, to a larger extent, the vast majority of forums on the internet to enable rich text formatting.
+It is made up of tags that surround the text to enrich and, sometimes, attributes.
+
+In the osu!website, BBCode is used in forum posts, signatures and userpages.
+This article is a listing of the available tags.
+
+## Notes
+
+Know that the osu!fourms gives you a few format buttons that you could use; however, you could just type them in yourself.
+
+BBCode tags are case insensitive.
+
+### BBCode Button Behavior
+
+If you highlight the text that you want to format then click on one of them, e.g. bold, italics, strikethrough, or underline, it will add said format code to the highlighted text automatically.
+
+If you click the button without highlighting text, it will surround the tags around your text cursor.
+
+### Combining tags
+
+You can combine tags to create an even richer formatting.
+
+When combining tags, you **must** respect the order/nesting of the tags.
+Failing to follow this may break your entire forum post, signature, or userpage.
+
+For example (pay attention to the order of the code):
+
+- `[centre]` `[b]` _text_ `[/b]` `[/centre]` is correct, but
+- `[b]` `[centre]` _text_ `[/b]` `[/centre]` is wrong.
+
+## List of BBCode Tags
+
+### Bold
+
+This will help emphasize words or an entire paragraph in a heavier way than [italics](#italic).
```

Comment:
`[italics](#italic)` 
Aren't hashtags case-sensitive?
(same in those ones below)